resources/language/resource.language.he_il/strings.po

```diff
@@ -864,7 +864,7 @@ msgstr "איפוס קוד ESN"
 
 msgctxt "#30218"
 msgid "Are you sure you want to reset the ESN?\r\nIf you confirm, wait until you receive the login message."
-msgstr "האם לאפס את קוד הESN בוודאות? \r\n במידה וכן, יש ללחוץ על הכפתור ולהמתין עד לקבלת מסך ההתחברות"
+msgstr "האם לאפס את קוד הESN בוודאות?\r\nבמידה וכן, יש ללחוץ על הכפתור ולהמתין עד לקבלת מסך ההתחברות"
```

Comment:
Can you check this translation? after my changes github report last word changed i do not know why

Reply:
Very strange, but it seems that since the space was omitted, the word "במידה" and the \n somehow parsed together, which is later read as "nבמידה" 
This also reproduces on PyCharm, see below screenshot

<img width="698" alt="image" src="https://user-images.githubusercontent.com/47418316/89729390-e9e23e00-da3d-11ea-8ca7-64cb6a441c4f.png">

I think keeping the space is the only way

Reply:
Can you try replace \r\n with [CR]
always leaving no spaces around?

Reply:
Will change and test locally 👍

Reply:
i was thinking another thing, is your language reading reversed Right to left?
if so, it is possible that \r\n go after `במידה`
but is better test on Kodi what happen actually
you can test by open Add-on settings, and in expert section, select Reset ESN, and the message will appear

Reply:
Tomorrow will do the release, i would like include your translation,
so if you have not time to check/fix this you can do it later time

Reply:
Fixed
<img width="806" alt="image" src="https://user-images.githubusercontent.com/47418316/89957253-9a6b6000-dc3f-11ea-85df-5caed8a018c4.png">